addons/docs/jest-transform-mdx.js

```diff
@@ -29,6 +29,10 @@ module.exports = {
       import { mdx } from '@mdx-js/react'
       ${mdx.sync(src, { compilers, filepath: filename })}
     `;
-    return getNextTransformer(filename, config).transformSource(filename, result, instrument);
+    
+    const extension = path.extname(filename);
+    const jsFileName = `${filename.slice(0, -extension.length)}.js`;
```

Comment:
```suggestion
    const jsFileName = `${filename}.js`;
```

Reply:
WDYT about this?

Reply:
Shouldn't we also update https://github.com/storybookjs/storybook/blob/c6d4f69d2895e063acdf6c4abdeb762fa1142a26/addons/docs/jest-transform-mdx.js#L12 then? Not sure if it'll help though

Reply:
Looks like this change is what actually breaks the tests. `foo.stories.mdx.js` doesn't match anything from our `transform` patterns in `jest.config.js`

Reply:
ahhh so i had it right the first time?  :)